app/models/npq_application.rb

```diff
@@ -79,6 +79,14 @@ class NPQApplication < ApplicationRecord
   delegate :id, :name, to: :npq_course, prefix: true
   delegate :id, :name, to: :npq_lead_provider, prefix: true
 
+  def current_contract
```

Comment:
contracts are provider specific, and tied to statements, we need to think about where we could query this, and if this should be on application or on the npq provider/statement instead, happy to chat through this